trajopt/src/problem_description.cpp

```diff
@@ -683,7 +684,7 @@ void CartVelTermInfo::fromJson(ProblemConstructionInfo& pci, const Json::Value&
 
 void CartVelTermInfo::hatch(TrajOptProb& prob)
 {
-    unsigned int n_dof = prob.GetKin()->numJoints();
+  unsigned int n_dof = prob.GetKin()->numJoints();
 
   if (term_type == (TT_COST | TT_USE_TIME))
```

Comment:
```suggestion
   if (term_type == (TT_COST  +  TT_USE_TIME))
```

Reply:
These should be equivalent as well. 
https://stackoverflow.com/questions/7334832/are-addition-and-bitwise-or-the-same-in-this-case

Reply:
That being said, if you'd rather me go with + instead of |, I can change them all to be that way. I've just always seen people use masks with and/or